src/redux/books/booksSlice.js

```diff
@@ -1,6 +1,25 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = [];
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 
 const booksSlice = createSlice({
   name: 'books',
```

Comment:
- [ ] I noticed you are still mutating the state by using the push keyword, I will encourage you to take advantage of the spread operator because it is not professionally advisable. You can read more on it [here](https://www.bam.tech/en/article/what-would-happen-if-you-mutated-your-react-redux-state).

So instead of `addBook: (state, action) => {
      state.push(action.payload);
    },`

You can make it look like `addBook: (state, action) => {
      state.books = [...state.books, action.payload]
    },`